ddtrace/tracer/span.go

```diff
@@ -337,6 +337,21 @@ func (s *span) String() string {
 	return strings.Join(lines, "\n")
 }
 
+func (s *span) Format(f fmt.State, c rune) {
+	switch c {
+	case 's':
+		fmt.Fprint(f, s.String())
+	case 'v':
```

Comment:
We should maybe have a default case, just in case... Otherwise printing the span with any other `c` will just result in an empty confusing string. Can we do something [like the standard library](https://play.golang.org/p/vQc2f1OIBJf) ?

```
default:
    fmt.Fprintf(f, "%%!%c(ddtrace.Span=%v)", c, s)
```

What do you think?

Reply:
I like the standard library solution. That way it remains consistent with what people expect.